websock/frame.nim

```diff
@@ -205,6 +205,9 @@ proc decode*(
 
   frame.length = finalLen
 
+  if frame.length > WSMaxMessageSize:
```

Comment:
A too big frame.length (of instance 13402712495349628929) causes issues in recv, capping it here